### PR TITLE
Fixed QuickSOrt

### PR DESCRIPTION
**Pull request by ahrenswett**

### CODE REVIEW

File: src/main/java/code401challenges/QuickSort.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 package code401challenges;
-
+//help From geeks for geeks
 public class QuickSort {
     public static int[] quickSort(int[]arr, int left,int right) {
```
Comment (jeffborda) [inline src/main/java/code401challenges/QuickSort.java:4]: You could also write you public method signature like: `public static int[] quickSort(int[]arr)` so that users don't have to specify what part of the array they want to sort.  Then have a helper method that takes in the left and right.

File: src/test/java/code401challenges/QuickSortTest.java [hunk 2]
```diff
@@ -11,7 +13,8 @@
     public void quickSortTest() {
         int[] test = {4,5,1,6,7,4,2,9,3,6};
         int[] ans = {1,2,3,4,4,5,6,6,7,9};
-        assertArrayEquals(ans, quickSort(test, test[0], test[test.length-1]));
+        System.out.println(Arrays.toString(quickSort(test, 0, test.length-1)));
```
Comment (jeffborda) [inline src/test/java/code401challenges/QuickSortTest.java:16]: This is a good test... but you have to test some edge cases to verify your method is working the way you want. Edge cases like: length 0 array, length 1 array, a pre-sorted array, and reverse sorted array.
